app/controllers/known_issues_controller.rb

```diff
@@ -10,24 +9,28 @@ def index
   # GET /known_issues/1
   def show
     @known_issue = KnownIssue.find(params[:id])
+    authorize @known_issue
```

Comment:
quick note because i just randomly clicked on this. you can do this which is shorter:
```suggestion
    @known_issue = authorize KnownIssue.find(params[:id])
```